### PR TITLE
Add ARM build to release script

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Should make the release process even easier.

### CODE REVIEW

**Comment by jonny-improbable** [comment]
See https://github.com/improbable-eng/grpc-web/pull/406; as mentioned in that thread, you can't cross compile for ARM on a macOS host, which, unfortunately, is what I use to build and publish releases.

We should either
1. Build release binaries a docker container
2. Automate the entire release process through a CI job

Option 1 is certainly simpler (:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,5 +74,6 @@ This is your prompt to open a web browser on https://localhost:9876 at which poi
 1. From a fresh checkout of master, create a release branch, ie: `feature/prepare-x.y.z-release`
 2. Update `CHANGELOG.md` by comparing commits to master since the last Github Release
 3. Raise a pull request for your changes, have it reviewed and landed into master.
-4. Switch your local checkout back to the master branch, pull your merged changes and run `./publish-release.sh`.
-5. Create the ARM binaries and attach them to the Github release.
\ No newline at end of file
+4. Make the release on github using the new commit in master.
```
Comment (jonny-improbable) [inline CONTRIBUTING.md]: The `publish-release.sh` script takes care of creating the github release and attaching the generated binaries to it. It also publishes the client(s) to npm.

Comment (johanbrandhorst) [reply CONTRIBUTING.md]: whew I clearly misunderstood the depth of this script.

**Comment by johanbrandhorst** [comment]
Ah, doh! I see. Nevermind this. I set up a auto publishing job for the grpc-gateway but we were using CircleCI. Might be possible to copy some of that though: https://github.com/grpc-ecosystem/grpc-gateway/blob/master/.circleci/config.yml#L77

**Pull request closed without merging**